DeviceDiagnostics/DeviceDiagnostics.cpp

```diff
@@ -264,14 +259,14 @@ namespace WPEFramework
         uint32_t DeviceDiagnostics::getVideoDecoderStatus(const JsonObject& parameters, JsonObject& response)
         {
             LOGINFOMETHOD();
-            response["video"] = getMostActiveDecoderStatus("video");
+            response["videoDecoderStatusChange"] = getMostActiveDecoderStatus("video");
```

Comment:
the param should be "videoDecoderStatus". The param in the event should be "videoDecoderStatusChange".